Cargo.toml

```diff
@@ -188,8 +188,9 @@ missing_panics_doc = "allow"
 module_name_repetitions = "allow"
 must_use_candidate = "allow"
 similar_names = "allow"
+single_match_else = "allow"
 too_many_lines = "allow"
-# To allow `#[allow(clippy::all)]` in `crates/ruff_python_parser/src/python.rs`.
+# Without the hashes we run into a `rustfmt` bug in some snapshot tests, see #13250
 needless_raw_string_hashes = "allow"
```

Comment:
We don't have a `python.rs` file anymore, but it still seems like a bad idea to enable this lint while the `rustfmt` bug we encountered in #13250 is a thing. So I just updated the comment.